### DISCUSSION TITLE
Why the provided "demo" example extends users::Model but not users::Entity?

### DISCUSSION THREAD

**Discussion by vrozhdestvenskyi** [loco-rs/loco#626] [Q&A]
Hi all and thank you very much for the great framework!

I've been looking through the Models code at the provided "demo" example and noticed that all the `find*`, `create*`, `set*` and alike methods extend `Model` structs (see e.g. [examples/demo/src/models/users.rs](https://github.com/loco-rs/loco/blob/master/examples/demo/src/models/users.rs#L87)). However, I would expect them to belong to `Entity`, mainly because of the following 2 reasons:
1. first, to comply with SeaORM design, and
2. second, these methods are static (i.e. they do not need any calling `Model` object)

My question is, why have you preferred such a design?

I'm just trying to use the framework in a proper way. Thanks in advance!

**Comment by kaplanelad** [comment]
@vrozhdestvenskyi see this seaorm [guid](https://www.sea-ql.org/SeaORM/docs/generate-entity/entity-structure/)

**Comment by vrozhdestvenskyi** [comment]
Thank you for getting back to me!
Sorry but I don't understand, how this guide is supposed to answer the question? Could you please elaborate a bit?
